openmoc/compatible/opencg_compatible.py

```diff
@@ -753,8 +768,8 @@ def get_openmoc_lattice(opencg_lattice):
   universes = opencg_lattice.universes
 
   # Initialize an empty array for the OpenMOC nested Universes in this Lattice
-  universe_array = np.ndarray(tuple(np.array(dimension[0:2])), \
-                              dtype=openmoc.Universe)
+  # The Lattice universe array is stored x-y-z in OpenCG, but z-y-x in OpenMOC
```

Comment:
Looking at the master branch of OpenCG, it appears OpenCG stores the Lattice universe array in z-y-x, just like in OpenMOC. For instance, line 1045 of `opencg/universe.py` indexes the universes array with the z-component first:

    return self._universes[lat_z][lat_y][lat_x]

Is OpenCG indeed storing the universes array as x-y-z?